### PR TITLE
Fix description of http.server.duration and http.server.request.duration

### LINKED ISSUE #2753: `http.server.duration` metric description is wrong
### Describe your environment

OS: K8S
Python version: 3.11
Package version: 0.46b0

### What happened?

our Otel Collectors are dropping metrics caused due to a description in some metrics not being set correctly, this was fine in 0.45b0, it seems that it was reverted/overridden [here](https://github.com/open-telemetry/opentelemetry-python-contrib/compare/v0.45b0...v0.46b0#diff-2c109e76591442fd2a0e5ac717f0148877d9e99cbc76619e4876942c9c86543dR551)
as a result we see the errors like in this [issue](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2298)

### Steps to Reproduce

1. set up otel collectors in k8s
2. setup fastAPI apps with auto-instrumentation 

### Expected Result

no errors 

### Actual Result

metrics being dropped as result of mismatch in description 

### Additional context

_No response_

### Would you like to implement a fix?

None

### PR DESCRIPTION
**Pull request by StevenPenn**
# Description
Currently, the description of "http.server.duration" and "http.server.request.duration" is not inconsistent with opentelemetry offical document.
https://opentelemetry.io/docs/specs/semconv/non-normative/http-migration/#http-server-duration-metric

Fixes #2753
https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2753

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
Test it in aiohttp-server, asgi, django, falcon, flask, pyramid, tornado, wsgi

# Does This PR Require a Core Repo Change?
- [x] No.

# Checklist:
See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.
- [x] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Review by lzchen — APPROVED** [review]

**Comment by lzchen** [comment]
@StevenPenn 

Please add a CHANGELOG entry, rebase to the latest main and also enable maintainer edits so we can rebase for you!

File: instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py [hunk 1]
```diff
@@ -571,14 +571,14 @@ def __init__(
             self.duration_histogram_old = self.meter.create_histogram(
                 name=MetricInstruments.HTTP_SERVER_DURATION,
                 unit="ms",
-                description="measures the duration of the inbound HTTP request",
+                description="Measures the duration of inbound HTTP requests.",
```
Comment (emdneto) [inline instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py:574]: Not a big deal but looking at [v1.11.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.11.0/specification/metrics/semantic_conventions/http-metrics.md#http-server) the description seems correct. wdyt @lzchen ?

Comment (lzchen) [reply instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py:574]: I saw that too but the link to the OT specs migration on the website seems more "current" -> https://opentelemetry.io/docs/specs/semconv/non-normative/http-migration/#http-server-duration-metric. I don't have a strong opinion though.

**Review by emdneto — APPROVED** [review]

**Review by xrmx — APPROVED** [review]

**Review by ocelotl — APPROVED** [review]

**Comment by ocelotl** [comment]
@StevenPenn please allow maintainers to edit your PR so that we can rebase it on `main`. If not anytime something else gets merged in `main` will require you to do this manually, which makes it very hard for us to merge your PR even with all tests passing.

**Comment by StevenPenn** [comment]
Hi @lzchen, I've updated the change log file.

**Comment by StevenPenn** [comment]
Hi @ocelotl @lzchen, due to the limitation, I am not required to enable maintainer edits. I'll keep track of this PR. Sorry for that.

**Comment by StevenPenn** [comment]
Hi @ocelotl @lzchen @xrmx @emdneto, I think we can merge this PR, thanks!

**Comment by lzchen** [comment]
@StevenPenn 

Unfortunately every time we merge another change you will have to rebase.

**Pull request merged**